src/components/AddTodoForm.js

```diff
@@ -1,14 +1,35 @@
-import React from "react";
+import React, { useState } from "react";
+import TodoList from "./TodoList";
 
 const AddTodoForm = () => {
+  const [TodoItem, setTodoItem] = useState("");
+  // By Bellow code we take the value of input feild and set it back to TodoItem using useState.
```

Comment:
It's tempting to manage state here but you'll find that it is very difficult to communicate new todos to the TodoList if you do this. State should be managed at the top of the component tree in App.